db/schema.rb

```diff
@@ -615,7 +615,6 @@
     t.string "code"
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
-    t.boolean "sync_courses", default: false, null: false
```

Comment:
can you commit the `ActiveRecord::Schema.define(version: ******) do` change as well?